README.md

```diff
@@ -1254,3 +1254,177 @@ docker run --net=host --entrypoint python3 -v /scality/ssd01/s3/scality-utapi/co
+    {
+      "arn": "arn:aws:iam::024022147664:/test_1669748782/",
+      "name": "test_1669748782",
+      "obj_count": 25,
+      "bytes_stored": 25,
+      "bytes_stored_total": 35
+    }
+  ],
+  "bucket": {
+    "arn:aws:iam::081797933446:/test_1669749866/": [
+      {
+        "name": "test3",
+        "obj_count": 25,
+        "bytes_stored": 25,
+        "bytes_stored_total": 35
+      }
+    ],
+    "arn:aws:iam::024022147664:/test_1669748782/": [
+      {
+        "name": "test",
+        "obj_count": 15,
+        "bytes_stored": 15,
+        "bytes_stored_total": 21
+      },
+      {
+        "name": "test2",
+        "obj_count": 10,
+        "bytes_stored": 10,
+        "bytes_stored_total": 14
+      }
+    ]
+  },
+  "service": {
+    "obj_count": 50,
+    "bytes_stored": 50,
+    "bytes_stored_total": 70
+  }
+}
+```
+
+### Authentication
+
+By default a random API key is generated and logged to stdout at startup.
+
+```json
+{"name":"s3utils::utapi-service-sidecar","time":1669677797644,"key":"c489cc0f5bec1c757be7aecd7cdd61dc9db5bfbc4e4f7bad","level":"info","message":"using random API key","hostname":"f2988d47e450","pid":1}
+```
+
+A custom static key can be set using the `SIDECAR_API_KEY` environment variable.
+
+```shell
+docker run -d \
+  --network=host \
+  -e SIDECAR_API_KEY=dev_key_change_me \
+  scality/s3utils service-level-sidecar/index.js
+```
+
+API requests to `/api/report` require this API token to be present in the `Authorization` header.
+
+```shell
+curl -X POST -H "Authorization: Bearer dev_key_change_me" localhost:24742/api/report
+```
+
+### Scale Factor
+
+Scale factor is provided as a floating point number that represents the percentage to scale reported byte values.
+For example, a value of `1.4` will output byte numbers 140% of the calculated value ie 100 bytes will be reported as 140 bytes.
+If this produces a non-integer result it is rounded up to the next whole number.
+
+It can be set using the `SIDECAR_SCALE_FACTOR` environment variable.
+
+```shell
+docker run -d \
+  --network=host \
+  -e SIDECAR_SCALE_FACTOR=1.4 \
+  scality/s3utils service-level-sidecar/index.js
+```
+
+### TLS
+
+TLS can be enabled for the api server as well as internal connections to vault and bucketd.
+To enable TLS for the API server specify the path to the TLS certificate and key using
+the `SIDECAR_TLS_KEY_PATH` and `SIDECAR_TLS_CERT_PATH` environment variables.
+A CA can optionally be set using `SIDECAR_TLS_CA_PATH`.
+Certificates should be mounted into the container using a docker volume.
+
+```shell
+docker run -d \
+  --network=host \
+  -v $PWD/certs:/certs \
+  -e SIDECAR_TLS_CERT_PATH=/certs/cert.pem \
+  -e SIDECAR_TLS_KEY_PATH=/certs/key.pem \
+  -e SIDECAR_TLS_CA_PATH=/certs/key.pem \ // CA path is optional
+  scality/s3utils service-level-sidecar/index.js
+```
+
+To enable TLS for internal connections to vault or bucketd set `SIDECAR_VAULT_ENABLE_TLS=true` or `SIDECAR_BUCKETD_ENABLE_TLS=true` respectively.
```

Comment:
Yeah, I heard some mention of https being a (soft) requirement and I wasn't sure if the platforms this would be running on had internal TLS or if this would be separate. I figured I'd just err on the side being flexible.